README.md

```diff
@@ -211,64 +262,68 @@ Alternatively, refactor the code to use `<img>` tags.
 
 ```html
 <div style="background-image: url(...)">
-  <span class='visually-hidden'>description of background image</span>
+  <span class="visually-hidden">description of background image</span>
 </div>
 ```
 
-### SVG
+### <a name="svg"></a>SVG
 
+When possible SVG images are prefered over bitmap. Some uses use screen enlargers to better view the content; SVG images scale losslessly while bitmap images become pixelated.
```

Comment:
```suggestion
When possible use SVG images over bitmap images because they can be enlarged via page zoom without losing quality; SVG images scale losslessly while bitmap images become pixelated.
```